schema/gene_family.js

```diff
@@ -0,0 +1,27 @@
+import type { GraphQLFieldConfig } from "graphql"
+import { GraphQLObjectType, GraphQLString, GraphQLList } from "graphql"
+import Gene from "./gene"
+
+const GeneFamilyType = new GraphQLObjectType({
+  name: "GeneFamily",
+  fields: {
+    id: {
+      type: GraphQLString,
+    },
```

Comment:
Can this field be `null`? If not, define it as such with `new GraphQLNonNull(GraphQLString)`.